demos/classification_demo/README.md

```diff
@@ -92,7 +92,18 @@ It has the following format:
 
 Class index values must be in range from 0 to 1000. If you want to use "other" class, which is supported only by a small subset of models, specify it with -1 index.
 
-["Labels" file](./synset_words.txt) contains the list of human-readable labels, one line for each class.
+"Labels" file contains the list of human-readable labels, one line for each class.
+
+Please note that you should use [id_classes_15.txt](./id_classes_15.txt) labels file with the following models:
```

Comment:
I think the file name should be more explicit about the file's origins, such as `imagenet_2015_classes.txt`.